### PR TITLE
Drop Container Component Statuses

### PR DESCRIPTION
**Pull request by zakiva**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1485431 
Schema: https://github.com/ManageIQ/manageiq-schema/pull/79
Core: https://github.com/ManageIQ/manageiq/pull/16055
UI: https://github.com/ManageIQ/manageiq-ui-classic/pull/2246

### CODE REVIEW

**Comment by simon3z** [comment]
@zakiva IIRC there's a BZ to link, right?

**Comment by zakiva** [comment]
> @zakiva IIRC there's a BZ to link, right?

Just added it :)

File: app/models/manageiq/providers/kubernetes/container_manager/refresher_mixin.rb [hunk 1]
```diff
@@ -5,9 +5,7 @@ module ContainerManager::RefresherMixin
         KUBERNETES_ENTITIES = [
           {:name => 'pods'}, {:name => 'services'}, {:name => 'replication_controllers'}, {:name => 'nodes'},
           {:name => 'endpoints'}, {:name => 'namespaces'}, {:name => 'resource_quotas'}, {:name => 'limit_ranges'},
-          {:name => 'persistent_volumes'}, {:name => 'persistent_volume_claims'},
-          # workaround for: https://github.com/openshift/origin/issues/5865
-          {:name => 'component_statuses', :default => []}
+          {:name => 'persistent_volumes'}, {:name => 'persistent_volume_claims'}
```
Comment (cben) [inline app/models/manageiq/providers/kubernetes/container_manager/refresher_mixin.rb]: nit: do you mind keeping the trailing comma? (not everyone likes them, your call)

**Comment by moolitayer** [comment]
AFAIU this patch should be independant

**Comment by simon3z** [comment]
> AFAIU this patch should be safe to merge

@moolitayer feel free to merge (sync with @zakiva ) when ready if decoupled from the other PRs.

**Comment by zakiva** [comment]
@moolitayer yeah this should be safe to merge now, actually, it has to be merged before we remove the class/table in the main and schema repos. 
(UI also should be safe).

File: app/models/manageiq/providers/kubernetes/container_manager/refresher_mixin.rb (hunk 1 above)
Comment (zakiva) [reply app/models/manageiq/providers/kubernetes/container_manager/refresher_mixin.rb]: I don't like it :) 
(and it wasn't there in the first place)

**Comment by moolitayer** [comment]
:scissors: :fire: :scissors: :fire: :scissors:

**Pull request merged**